src/Agent.Plugins/GitSourceProvider.cs

```diff
@@ -231,6 +231,15 @@ public abstract class GitSourceProvider : ISourceProvider
             string clientCertPrivateKeyAskPassFile = null;
             bool acceptUntrustedCerts = false;
 
+            bool reducedOutput = StringUtil.ConvertToBoolean(
+                executionContext.Variables.GetValueOrDefault("agent.source.checkout.quiet")?.Value ??
+                System.Environment.GetEnvironmentVariable("AGENT_SOURCE_CHECKOUT_QUIET"), false);
```

Comment:
It seems kinda odd to me to have this as a job variable or env variable as opposed to an input to the checkout task. Any reason we went this route?

Reply:
Other than that, the implementation itself looks good to me

Reply:
Can designer builds take additional inputs to the checkout step? The initial request for this change came from a designer/TFVC customer.

Reply:
No, that makes sense, I'm good merging this.